weave/trait/telemetry/wdm/telemetry_wdm_trait.proto

```diff
@@ -0,0 +1,94 @@
+//
+//    Copyright (c) 2016 Nest Labs, Inc.
+//    All rights reserved.
+//
+//    This document is the property of Nest. It is considered
+//    confidential and proprietary information.
+//
+//    This document may not be reproduced or transmitted in any form,
+//    in whole or in part, without the express written permission of
+//    Nest.
+//
+
+//    @file
+//      This file specifies a WDM Telemetry trait that specifies the
+//      various telemetry counters that are associated with WDM
+
+syntax = "proto3";
+
+package weave.trait.telemetry.wdm;
+
+import "wdl/wdl_options.proto";
+import "google/protobuf/timestamp.proto";
+
+option java_outer_classname = "WeaveInternalTelemetryWdmTrait";
+option objc_class_prefix = "SCM";
+
+//  @brief
+//    TelemetryWdmTrait
+//
+//  A telemetry trait for the WDM that specifies various counters
+//  associated with WDM updates and subscriptions.
+//
+message TelemetryWdmTrait {
+  option (wdl.message_type) = TRAIT;
+  option (wdl.trait) = {
+    stability: ALPHA,
+    id: 0x1781,
+    vendor_id: 0x0000,
+    version: 1
+  };
+  option (wdl.properties) = {
+    writable: READ_ONLY,
+  };
+
+  // Events associated with WDM updates
+  message TelemetryWdmUpdateStatsEvent {
+    option (wdl.message_type) = EVENT;
+    option (wdl.event) = {
+      id: 1
+      event_importance: EVENT_IMPORTANCE_DEBUG
+    };
+
+    // This field holds the count of successful updates since the
+    // last restart.
+    uint32 successful_updates_count = 1;
```

Comment:
Are these metrics aggregated across different subscriptions?

Reply:
Yes this is accumulated across multiple subscriptions.

Reply:
So, this looks like an aggregate of all Updates(presumably unconditional and conditional). It might be good to separate conditional vs unconditional updates also?